core/kernel/persistence/smoothsql/class.SmoothRdf.php

```diff
@@ -107,13 +106,17 @@ public function addTripleCollection(iterable $triples)
     protected function insertTriples(array $triples)
     {
         $values = array_map([$this,"tripleToValue"], $triples);
-        $isInsertionSuccessful = $this->insertValues($values);
-        return $isInsertionSuccessful;
+        return $this->insertValues($values);
     }
 
     protected function insertValues(array $valuesToInsert)
     {
-        return $this->getPersistence()->insertMultiple('statements', $valuesToInsert);
+        $types = [];
+        foreach ($valuesToInsert as $value) {
```

Comment:
- [ ] Fix the expression

$value is never used

Reply:
nope but need to loop to accumulate type foreach value